pkg/virt-controller/watch/pool.go

```diff
@@ -1217,7 +1221,7 @@ func (c *PoolController) updateStatus(origPool *poolv1.VirtualMachinePool, vms [
 	pool.Status.Replicas = int32(len(vms))
```

Comment:
we probably need a check somewhere to see if we've reached the number of ready replicas